src/usdt.cpp

```diff
@@ -129,10 +143,13 @@ void USDTHelper::read_probes_for_pid(int pid)
     void *ctx = bcc_usdt_new_frompid(pid, nullptr);
     if (ctx == nullptr)
     {
-      LOG(ERROR) << "failed to initialize usdt context for pid: " << pid;
+      if (print_error)
```

Comment:
Added this because the error logs are very noisy when targeting all processes on a host.